submissions/Semka094/hooli-style-popup/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Jost:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap"
+          rel="stylesheet">
+    <title>Pop-up</title>
+</head>
+
+<body>
+
+<div class="menuContainer">
+    <nav class="navWrapper">
+        <div class="navItem">
+            <a class="navLink" href="#">Mail</a>
+        </div>
+        <div class="navItem">
+            <a class="navLink" href="#">Images</a>
+        </div>
+        <div class="navItem icon">
```

Comment:
Fix mobile view
![image](https://user-images.githubusercontent.com/16196199/187025446-4ba8ebd5-8a23-44c3-8246-73dc90338ea9.png)

Reply:
Fixed.
I haven't seen in the requirement to implement mobile version that is why it wan't implemented initially.